### PR TITLE
Minor refactoring

### PR DESCRIPTION
**Pull request by chrisccerami**
Just looking around the source and I saw a few things I thought could be touched up.

### CODE REVIEW

File: addon/components/zoom-zone.js [hunk 1]
```diff
@@ -53,18 +53,15 @@ export default Component.extend({
...
+    this.get('$viewport').off();
+    this.get('$content').off();
 
     this._super(...arguments);
   },
 
   matrix: computed('panX', 'panY', 'scale', function () {
-    const [scale, x, y] = [this.get('scale'), this.get('panX'), this.get('panY')];
-    return `matrix(${scale}, 0, 0, ${scale}, ${x}, ${y})`;
+    const { scale, panX, panY } = this;
```
Comment (crazymykl) [inline addon/components/zoom-zone.js]: I'm not certain accessing these directly won't cause issues if other code uses observers on them. Maybe use `getProperties`?

Comment (chrisccerami) [inline addon/components/zoom-zone.js]: Hm, what do you suspect might be the problem(s) with Observers on these fields?

Comment (crazymykl) [inline addon/components/zoom-zone.js]: If there is something observing when they are read, it wont fire if they are accessed without the use of `get`.

Comment (chrisccerami) [inline addon/components/zoom-zone.js]: Interesting point. How would someone observe when a property is read? Is that an Ember internal property you'd be observing on?

**Pull request closed without merging**